src/modules/sponsor/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { getAccount, getToken, getWorkspace } from '../../conf'
+import * as env from '../../env'
+import envTimeout from '../../timeout'
+import userAgent from '../../user-agent'
+
+
+const DEFAULT_TIMEOUT = 10000
+
+export const options = {
+  timeout: (envTimeout || DEFAULT_TIMEOUT) as number,
+  retries: 3,
+}
+
+export const getIOContext = () => ({
```

Comment:
Wasn't this declared anywhere else before? 🤔

Reply:
I don't think so, not exactly like this. But it would be good to put this in a more global `utils.ts` and use it elsewhere in the code. It would clean up some other parts of the code. Will do it in later PRs.